ssifwc/precipitation.py

```diff
@@ -49,15 +49,15 @@ def _get_precipitation_for_year(self, year):
                         day = f'0{day}'
                     value = float(row.split(',')[i + 1].strip())
                     if value == -99:
-                        value = 0
+                        value = None
```

Comment:
I've changed -99 from `0` to `None` - this is so the charting library we use on the frontend will remove these points from the graph making it less messy.